tests/delete/tests.py

```diff
@@ -503,3 +503,12 @@ def test_fast_delete_large_batch(self):
         # that + fast delete of the related objs.
         self.assertNumQueries(2, a.delete)
         self.assertEqual(User.objects.count(), 0)
+
+    def test_fast_delete_empty_no_update_can_self_select(self):
+        """
+        #25932 - Fast deleting on backends that don't have the
+        `no_update_can_self_select` feature should work event if the specified
```

Comment:
Should we also add an assertion to demonstrate that the query doesn't match any row?